### DISCUSSION TITLE
Count not reflected within Model / Count not queryable

### DISCUSSION THREAD

**Discussion by valoricDe** [MichalLytek/typegraphql-prisma#131] [Q&A] [answered]
Hi @MichalLytek in changelog for 0.13.1 you mention that the count is now selectable (only for relations?) and also in the generated resolvers I see

```
...
const { _count } = transformFields(
      graphqlFields(info as any)
    );
...
...(_count && transformCountFieldIntoSelectRelationsCount(_count)),
```

But the generated models do not contain the _count fields. As it is currently not possible to use the "extends type" functionality of graphql I'm wondering how it is possible to query `_count`

**Comment by MichalLytek** [comment, marked as answer]
As stated in changelog, you need to use `selectRelationCount` preview feature.

**Reply by valoricDe** [reply]
Ahh ok I was not so familiar with prisma in detail yet. Thank you for the pointer. So for other you have to:

```
generator client {
  provider = "prisma-client-js"
  previewFeatures = ["selectRelationCount"]
}
```
